### PR TITLE
Fix a color issue - sites open for both distribution and donation

### PR DESCRIPTION
**Pull request by gabe-bergstrom**
issue: https://github.com/jdalt/twin-cities-aid-distribution-locations/issues/10

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -196,7 +196,7 @@
               const accessibleColorTransform = { // this way you don't have to change everything in the google spreadsheet
                 "#fc03df": "#2c7bb6",
                 "#03bafc": "#abd9e9",
-                "#9f48ea": "#fdae61",
+                "#9F48EA": "#fdae61",
```
Comment (Olson3R) [inline index.html]: Should we do this or convert to lower case when we use this mapping?

Comment (maxine) [reply index.html]: Thanks for catching. I think it makes sense to use `toLowerCase()` when using the mapping to make sure everything is standard.

**Review by maxine — CHANGES_REQUESTED** [review]
`const color = item.gsx$color.$t.toLowerCase()`
then in the marker/sidebar element creators replace `accessibleColorTransform[item.gsx$color.$t]` with `accessibleColorTransform[color]`

**Review by maxine — CHANGES_REQUESTED** [review]
Change still needs to be made on line 210 to update the marker color
`location.marker = new mapboxgl.Marker({ color: accessibleColorTransform[color] })`

**Comment by maxine** [comment]
@gabe-bergstrom one more spot where this needs to be updated (L. 210).

**Pull request closed without merging**